README.md

```diff
@@ -46,14 +46,33 @@
 
 # OpenTelemetry Collector Contrib
 
-This is a repository for OpenTelemetry Collector contributions that are not
-part of the [core
-distribution](https://github.com/open-telemetry/opentelemetry-collector) of the
-Collector. Typically, these contributions are vendor specific
-receivers/exporters and/or components that are only useful to a relatively
-small number of users.
-
-> Please note that this repository and its releases are a superset of the core repository.
+This is a repository for OpenTelemetry Collector components that are not suitable for the  [core repository](https://github.com/open-telemetry/opentelemetry-collector) of the collector. Some of these components are part of the core distribution, such as the Jaeger and Prometheus components, but most of the components here are only available as part of the "contrib" distribution. Users of the OpenTelemetry Collector are also encouraged to build their own custom distributions with the [OpenTelemetry Collector Builder](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder), using the components they need from the core repository, the contrib repository, and possibly third-party or internal repositories.
```

Comment:
> Some of these components are part of the core distribution, such as the Jaeger and Prometheus components

Not sure if I read it right but Jaeger and Prometheus were moved from core to contrib quite a while ago

Reply:
Thats true, but that are still included in the core distribution of the collector: https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol/manifest.yaml

Reply:
So what I'm thinking is that perhaps this would be more clear in what's the relationship between core/contrib repositories/releases

```suggestion
This is a repository for OpenTelemetry Collector components that are not suitable for the  [core repository](https://github.com/open-telemetry/opentelemetry-collector) of the collector. 

The official distributions, core and contrib, are available as part of the [opentelemetry-collector-releases](https://github.com/open-telemetry/opentelemetry-collector-releases) repository. Some of the components in this repository are part of the "core" distribution, such as the Jaeger and Prometheus components, but most of the components here are only available as part of the "contrib" distribution. Users of the OpenTelemetry Collector are also encouraged to build their own custom distributions with the [OpenTelemetry Collector Builder](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder), using the components they need from the core repository, the contrib repository, and possibly third-party or internal repositories.
```